# PDB_FoldbackIntercoil

Foldback Intercoil (FBI), guanine-quadruplex (G-Quad.), and paranemic helix (PX) DNA Nanostructures have four-stranded DNA configurations that are geometrically distinct. Intercoil (intertwining DNA duplexes sharing the same helix axis, such as FBI) and supercoil (winding DNA duplexes without sharing their helix axes, such as G-Quad. and PX) structures are formed by a twist of the DNA double helix.[1], [2] Although the G-Quad. (supercoil) with its functions and the PX-DNA Nanostructure (supercoil) have been extensively studied and have been synthetically constructed, researchers have not yet intensively studied Intercoil DNA Nanostructures and have rarely tried to construct them due to the difficulties in finding and purifying intercoiled DNA structures from biological species and in designing structures with two DNA duplexes sharing the same helical centers. However, FBI DNA Nanostructures have unique geometrical features with important functionalities that are closely related to various in vivo phenomena, such as transposition, inversion, deletion, and insertion. To be specific, this compact four-stranded DNA form has long been suggested to explain and reconcile discrepancies in unusual biological phenomena (i.e., fast unwinding during replication, small-sized target site duplication (5-7 base pairs), and chromosomal pairing and synapsis without double-strand break enzymes) which are relatively hard to explain using canonical B-form duplexes.[3], [4], [5]


To construct the coordinate information of Foldback Intercoil motif, the suggested sequence of 39 base-pair of B-form DNA was generated [42]. Those B-form DNA is composed of three regions; loop, stem, and tail. After the standard B-form DNA was constructed, it was unwounded that the helicity was removed [Figure 24 (a)]. Second, base-pairs in the loop region were bended to the point that the distance between interfacing base-pairs next to the loop region is distanced about ~2.2 Å [Figure 24 (b)]. Once reached, two base-pairs of each termini of the loop section were symmetrically rotated into the opposite direction until facing base-pair in the stem section is parallelized [Figure 24(c). This parallelization slightly increase the distance between interfacing base-pairs in the stem region, so overall process of loop bending and stem parallelization was repeated until the distance between the interfacing base-pairs of the stem region is reasonably close enough, which means atoms in each base-pairs are not overlapping [Figure 24 (d)]. Stem section of the base-pairs was rotated about z-axis again to form intercooled helical structure of the Foldback intercoil DNA motif [Figure 24 (e)]. Rest of the bases were bended and rotated accordingly to form tail region of typical flexible B-form duplex DNA [Figure 24 (f)].


![alt text](fbi-process.png?raw=true "Reconstruction process of FoldbackIntercoil DNA Motif.")

### **References**
##### _[1]	G. Biffi, D. Tannahill, J. McCafferty, and S. Balasubramanian, “Quantitative Visualization of DNA G-quadruplex Structures in Human Cells,” Nat. Chem., vol. 5, no. 3, pp. 182–186, Mar. 2013._
##### _[2]	X. Wang, X. Zhang, C. Mao, and N. C. Seeman, “Double-stranded DNA homology produces a physical signature,” Proc. Natl. Acad. Sci., vol. 107, no. 28, pp. 12547–12552, Jul. 2010._
##### _[3]	B. D. Kim, “Four-stranded DNA: an intermediate of homologous recombination and transposition,” Korean J Breed, no. 17, pp. 453–466, 1985._
##### _[4]	P. A. W. Edwards, “A sequence-independent, four-stranded, double Watson-Crick DNA helix that could solve the unwinding problem of double helices,” J. Theor. Biol., vol. 70, no. 3, pp. 323–334, 6 1978._
##### _[5]	S. McGavin, “Models of specifically paired like (homologous) nucleic acid structures,” J. Mol. Biol., vol. 55, no. 2, pp. 293–298, 28 1971._


